classes/locations.php

```diff
@@ -60,18 +61,20 @@ public function register_locations() {
 			'rewrite'              => false,
 			'query_var'            => 'sm-location',
 			'register_meta_box_cb' => array( &$this, 'location_meta_cb' ),
-			'supports'             => array(),
+			'supports'             => array( 'title','author', 'editor'),
 			'labels'               => array(
-				'name'               => 'Locations',
+				'name'               => 'SimpleMap',
 				'singular_name'      => 'Location',
 				'add_new_item'       => 'Add New Location',
+				'all_items'         => 'Locations',
 				'edit_item'          => 'Edit Location',
 				'new_item'           => 'New Location',
 				'view_item'          => 'View Locations',
 				'search_items'       => 'Search Locations',
 				'not_found'          => 'No Locations found',
 				'not_found_in_trash' => 'No Locations found in trash',
 			),
+			'menu_icon'             => 'dashicons-location-alt',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->